addons/test_discuss_full/static/tests/livechat_session.test.js

```diff
@@ -0,0 +1,27 @@
+import { defineLivechatModels } from "@im_livechat/../tests/livechat_test_helpers";
+
+import { contains, openDiscuss, start, startServer } from "@mail/../tests/mail_test_helpers";
+
+import { describe, test } from "@odoo/hoot";
+
+import { Command, onRpc, serverState } from "@web/../tests/web_test_helpers";
+
+describe.current.tags("desktop");
+defineLivechatModels();
+
+test("should open a livechat channel", async () => {
+    const pyEnv = await startServer();
+    const guestId = pyEnv["mail.guest"].create({ name: "Visitor" });
+    const channel_id = pyEnv["discuss.channel"].create({
+        channel_type: "livechat",
+        channel_member_ids: [
+            Command.create({ partner_id: serverState.partnerId, livechat_member_type: "agent" }),
+            Command.create({ guest_id: guestId, livechat_member_type: "visitor" }),
+        ],
+        livechat_operator_id: serverState.partnerId,
+    });
+    onRpc("hr.employee", "search_read", () => [{ id: 90 }]);
```

Comment:
No fake result, either let it return nothing or add the actual data in env and ensure the mock server does its jobs.